### DISCUSSION TITLE
How to change the color of menu item when it is clicked

### DISCUSSION THREAD

**Discussion by bigbean17** [primefaces/community#1834] [PrimeVue]
Hi,

I'm trying to change the color menu items. Here is what I have for now
```
.p-menubar .p-menubar-root-list > .p-menuitem > .p-menuitem-content:hover,
.p-menubar .p-menubar-root-list > .p-menuitem > .p-menuitem-content:focus
{
    background-color: rgb(232, 142, 157);
    color: black;
}
```
This works when hover and click, the color changes as expected. But, after click the item, if I move my curse out of the item, it will change back to white. 
<img width="447" alt="image" src="https://github.com/primefaces/community/assets/69971426/3116aac8-9031-4025-b37d-b31155457a49">
The above image is when I click and keep my cursor on that item (home)
<img width="444" alt="image" src="https://github.com/primefaces/community/assets/69971426/be99d638-a970-4f5e-a3db-081e34cdab07">
The above image is when I click and move my cursor to another place, it supposed to be black

So what is the best way of changing that style so that the color only change from black to ping when I hover or click items? 

Thanks for helping

**Comment by FernandezLeandro** [comment]
Hi, I'm having the same problem, have you been able to solve it?
